website/src/pages/index.js

```diff
@@ -85,79 +79,59 @@ function Home() {
       <Layout permalink="/">
         <div
           className="container container--fluid home"
-          style={{ padding: "0", background: "#FFF" }}
+          style={{ padding: 0, background: '#FFF' }}
         >
-          <header className='baton-hero'>
-            <div className='container'>
+          <header className="baton-hero baton-hero--compact">
+            <div className="container">
               <div>
-                <h1 className="heading-1">The dbt Developer Hub</h1>
-                <p>Find everything you need to build, document, and collaborate with dbt — faster.</p>
-                <div 
-                  className="hero-search-input" 
-                  onClick={handleSearch}
-                  onKeyDown={(e) => {
-                    if (e.key === 'Enter' || e.key === ' ') {
-                      e.preventDefault();
-                      handleSearch();
-                    }
-                  }}
-                  role="button"
-                  tabIndex={0}
-                  aria-label="Open search"
-                >
-                  <input
-                    type="text"
-                    placeholder="⌘K to search"
-                    readOnly
-                    className="input"
-                    aria-hidden="true"
-                  />
-                </div>
-                <div className="quick-links">
-                  <span>Quick links</span>
-                  <Link to="/guides">Guides</Link>
-                  <Link to="/docs/dbt-versions/dbt-cloud-release-notes">Release notes</Link>
-                  <Link to="/docs/introduction">Product docs</Link>
-                </div>
-                <div className="hero-cta">
-                  <Link to="/docs/install-dbt-extension" class="primary-cta">Install Fusion + VS Code extension</Link>
+                <h1 className="heading-1 heading-1--tight">The dbt Developer Hub</h1>
+                <p className="hero-subcopy">
+                Find everything you need to build, document, and collaborate with dbt &mdash; faster.
+                </p>
+
+                {/* quickstarts moved up with tightened spacing */}
+                <div className="hero-cta hero-cta--tight">
+                  <Link className="hero-border-beam-cta" to="/docs/install-dbt-extension">
+                    <span>Install dbt VS Code extension + Unlock Fusion</span>
```

Comment:
i tried adding the `BorderBeam` component here but it didn't have the same effect. i wonder if it needed a rectangle shape? or maybe i needed to update the borderbeam component itself?